compatibility/sandbox-migration/runner/Migration/Runner.hs

```diff
@@ -102,6 +102,9 @@ main = do
 supportsSandboxOnX :: SemVer.Version -> Bool
 supportsSandboxOnX v = v == SemVer.initial || v >= (SemVer.initial & SemVer.major .~ 2)
 
+supportsSandboxOnXHocon :: SemVer.Version -> Bool
+supportsSandboxOnXHocon v = v == SemVer.initial
```

Comment:
Strange to see `==` here, was expecting `>=`

Reply:
I actually need help here, it should be >= {version this goes to `main`}
i.e. something like > `2.3.0-snapshot.20220619.10104.0.253b0b18`
do you know how to do that?